standalone-metastore/metastore-common/src/main/thrift/hive_metastore.thrift

```diff
@@ -2966,7 +2971,7 @@ PartitionsResponse get_partitions_req(1:PartitionsRequest req)
   bool submit_for_cleanup(1:CompactionRequest o1, 2:i64 o2, 3:i64 o3) throws (1:MetaException o1)
   void add_dynamic_partitions(1:AddDynamicPartitions rqst) throws (1:NoSuchTxnException o1, 2:TxnAbortedException o2)
   // Deprecated, use find_next_compact2()
-  OptionalCompactionInfoStruct find_next_compact(1: string workerId) throws(1:MetaException o1)
+  OptionalCompactionInfoStruct find_next_compact(1: string workerId, 2: string poolName) throws(1:MetaException o1)
```

Comment:
Hm, I'm not sure I understand why we need this change if we can use find_next_compact2?

Reply:
You are right, I reverted this one.